src/js/map/MapController.js

```diff
@@ -728,15 +728,15 @@ define([
 
                     brApp.map.infoWindow.show(mapPoint);
 
-                    // var centerPoint = new Point(brApp.map.extent.getCenter());
-                    // var centerPointScreen = brApp.map.toScreen(centerPoint);
-                    // if (centerPointScreen.y > 300) {
-                    //     centerPointScreen.y -= 100;
-                    // }
+                    var centerPoint = new Point(brApp.map.extent.getCenter());
```

Comment:
Why do you create a 'centerPoint' variable here and set it equal to a new Esri Point, when we are simply resetting that variable to a different point on 737 without ever having used it?